apps/service_providers/llm_service/main.py

```
@@ -166,7 +166,13 @@ def get_output_parser(self):
     def _default_parser(
         self, llm_output, session: ExperimentSession, include_citations: bool = True
     ) -> LlmChatResponse:
-        llm_outputs = llm_output.get("output", "")
+        if isinstance(llm_output, dict):
+            llm_outputs = llm_output.get("output", "")
+        elif isinstance(llm_output, str | list):
+            llm_outputs = llm_output
+        else:
+            raise ValueError(f"Unexpected type {type(llm_output)}")
```

Ruff (0.12.2)

Prefer `TypeError` exception for invalid type

(TRY004)